docs/monthly/2024-01-monthly.md

```diff
@@ -12,150 +12,97 @@ developments to gather their feedback on proposed plans.
 
-* Similarly, new feature to [run smoke tests in Sanchonet
-  #1257](https://github.com/input-output-hk/hydra/issues/1257) which will serve
-  as another proof point of being able to open/close heads in the new Conway
-  era.
+* Worked on implementing the new feature, [running smoke tests in SanchoNet #1257](https://github.com/input-output-hk/hydra/issues/1257), which will provide additional evidence of the capability to open and close heads in the Conway era.
 
-* Groomed and started work on [building and deploying a Hydra heads explorer
-  #696](https://github.com/input-output-hk/hydra/issues/696), details below.
+* Groomed and started work on [building and deploying a Hydra heads explorer #696](https://github.com/input-output-hk/hydra/issues/696), see details below.
 
-* Renamed "Drop Babbage support #1178" to [Switch L2 ledger to Conway
-  #1178](https://github.com/input-output-hk/hydra/issues/1178) to better capture
-  it's content.
+* Renamed 'Drop Babbage support #1178' to [Switch layer 2 ledger to Conway #1178](https://github.com/input-output-hk/hydra/issues/1178) to better capture its content.
 
 ## Hydra development
 
-[Issues and pull requests closed since the last
-report](https://github.com/input-output-hk/hydra/issues?q=is%3Aclosed+sort%3Aupdated-desc+closed%3A2023-11-30..2024-01-31)
+[Issues and pull requests closed since the last report](https://github.com/input-output-hk/hydra/issues?q=is%3Aclosed+sort%3Aupdated-desc+closed%3A2023-11-30..2024-01-31).
 
 This month, the team worked on the following:
 
 ### API changes: dropping JSON Tx
 
-The Hydra API now uses a TextEnvelope containing the upstream CBOR encoding
-from cardano-api for the `Tx` type and has removed the JSON representation from
-the API [#1240](https://github.com/input-output-hk/hydra/pull/1240). This is so
-transaction types have a fixed canonical encoding which will not change between
-versions. Consequently, consumers of the Hydra API will need to deserialise
-CBOR in order to inspect transactions.
+The Hydra API now utilizes a TextEnvelope containing the upstream CBOR encoding from `cardano-api` for the `Tx` type and removes the JSON representation from the API [#1240](https://github.com/input-output-hk/hydra/pull/1240). This ensures that transaction types have a fixed canonical encoding that will not change between versions. As a result, consumers of the Hydra API will need to deserialize CBOR to inspect transactions.
 
-This representation is equivalent to the TextEnvelope produced by cardano-cli,
-except with an additional `txId` field, that is guaranteed to match the transaction
-id within the cbor object.
+This representation mirrors the TextEnvelope produced by the Cardano CLI, with the addition of an `txId` field, which is guaranteed to match the transaction ID within the CBOR object.
```

Comment:
Suggestion: stick with `cardano-cli` to refer to the binary / tool

@olgahryniuk @oduameh

Reply:
Agreed 👍